site/src/utils/workspace.tsx

```diff
@@ -307,3 +312,23 @@ const FALLBACK_ICON = "/icon/widgets.svg";
 export const getResourceIconPath = (resourceType: string): string => {
 	return BUILT_IN_ICON_PATHS[resourceType] ?? FALLBACK_ICON;
 };
+
+export const lastUsedMessage = (lastUsedAt: string | Date) => {
+	const t = dayjs(lastUsedAt);
+	const now = dayjs();
+	let message = t.fromNow();
+
+	if (t.isAfter(now.subtract(1, "hour"))) {
+		message = "Now";
+	} else if (t.isAfter(now.subtract(3, "day"))) {
```

Comment:
Am I crazy or do these branches not do anything since message already defaults to `t.fromNow()` on line 319?